data-service/src/cucumberTest/java/cucumber/steps/CreateEnvironmentSteps.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package cucumber.steps;
+
+import cucumber.api.java8.En;
+import org.apache.commons.lang3.NotImplementedException;
+import org.springframework.test.context.ContextConfiguration;
+
+@ContextConfiguration(locations = "classpath:cucumber.xml")
```

Comment:
Is it necessary to fall back on XML config here? It looks like this should work just fine with `@ContextConfiguration(classes=com.amazonaws.blox.dataservice.Application.class)` instead.

Reply:
we need separate configuration for integ tests but I can move it into java config